app/AppKernel.php

```diff
@@ -20,6 +20,7 @@ public function registerBundles()
             new Knp\Bundle\MenuBundle\KnpMenuBundle(),
             new DashboardHub\Bundle\AppBundle\DashboardHubAppBundle(),
             new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
+            new DashboardHub\Bundle\DevelopmentBundle\DashboardHubDevelopmentBundle(),
```

Comment:
Ok - these are the magic things that Symfony added for me when I followed the instructions for creating a new Bundle, so please feel free to move.